encorelab/ck-board
# Issue 478 include the board as a workflow destination

### Overview

- Distribution workflow issues: destination to include one of the source items with multiple destinations; destination and source match the destination, error message created
- Ordering of dest/source Options: CANVAS before BUCKETS
- Merged with remote branch, added redis dump to gitignore
- Prioritized projects in source and destination lists; see TODOs to complete
- Allow for boards as destinations and changed BOARD to CANVAS
- Added comments to create workflow component and TODOs to ts and html files
- Added CANVAS and BUCKET identifiers to each of the destinations and sources
- Removed if statement: add CANVAS to destination for distribution workflow

**Test**

- Create distribution workflow with same destination and source (error message should occur)
- Create distribution workflow with multiple destinations (no error message)
- Ensure no console errors

Closes #478
